Sources/Core/Sources/Game.swift

```diff
@@ -9,6 +9,8 @@ public final class Game: @unchecked Sendable {
   public private(set) var tickScheduler: TickScheduler
   /// The event bus for emitting events.
   public private(set) var eventBus: EventBus
+  /// The struct that allows for accessing clientside configuration values.
+  public let configuration: ClientConfiguration
```

Comment:
Instead of storing ClientConfiguration in Game, pass it straight through to PlayerInputSystem (Game doesn't actually need the ClientConfiguration itself)